worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -123,7 +123,10 @@ namespace RTC
 
 		auto* storageItem = this->buffer[0];
 
-		delete storageItem;
+		// Reset (free RTP packet) the old storage item.
+		storageItem->Reset();
```

Comment:
@jmillan I think default destructor do same things, and we can remove ```Reset``` and use ```std::allocator_traits<Alloc>::destroy````

Reply:
Correct, Reset has the same body as the default destructor should have.